src/send/reducers.ts

```diff
@@ -41,30 +42,42 @@ export const sendReducer = (
   switch (action.type) {
     case REHYDRATE: {
       // Ignore some persisted properties
+      const now = Date.now()
       return {
         ...state,
         ...getRehydratePayload(action, 'send'),
         isSending: false,
         encryptedComment: null,
         isEncryptingComment: false,
+        // Keep only the last 24 hours recent payments
+        recentPayments: state.recentPayments.filter(
+          (p: PaymentInfo) => timeDeltaInHours(now, p.timestamp) < 24
+        ),
```

Comment:
this is just cleanup, seems like we should do this on rehydrate than on new payments succeeded